src/main/java/reposense/parser/ArgsParser.java

```diff
@@ -115,15 +120,19 @@ public static CliArguments parse(String[] args) throws ParseException {
 
             verifyDatesRangeIsCorrect(sinceDate, untilDate);
 
-            if (locations != null) {
-                return new LocationsCliArguments(locations, outputFolderPath, sinceDate, untilDate, formats);
+            if (reportFolderPath != null && !reportFolderPath.equals(EMPTY_PATH)) {
```

Comment:
not quite convinced by this... so if someone else type `-view .`, wouldn't it still runs the analysis since its pointing to CWD?

Reply:
I thought so too at first and did some testing. Typing `-view .` does not cause `reportFolderPath.equals(EMPTY_PATH) == True`. I tried other inputs as well and didn't find any that is equals to `Paths.get("")`.

From https://docs.oracle.com/javase/8/docs/api/java/nio/file/Paths.html#get-java.lang.String-java.lang.String...- explaining the `Paths.get()` method:

> A Path representing an empty path is returned if first is the empty string and more does not contain any non-empty strings.

From https://docs.oracle.com/javase/8/docs/api/java/nio/file/Path.html explaining empty Path:

> A Path is considered to be an empty path if it consists solely of one name element that is empty.

I think as long as there is an input, even if it is "." or cwd, the name element will not be empty. The documentation for `equals()` method of `Path` didn't specify exactly how `Path` instances are compared but it hints that that the hashCode is used. From https://docs.oracle.com/javase/8/docs/api/java/nio/file/Path.html#hashCode--:

> The hash code is based upon the components of the path

So I believe that any `Path` with non-empty name element will not be equals to an empty Path.

> Paths.get("") points to cwd

I believe this behaviour only occurs when accessing an empty path. From https://docs.oracle.com/javase/8/docs/api/java/nio/file/Path.html:

> Accessing a file using an empty path is equivalent to accessing the default directory of the file system.

Reply:
Basically it is true that Path.get("") points to CWD but I think Path.get("") is not equals to any non-empty path (eg. cwd or ".")

Reply:
I see. In that case, let's add another test to keep this assumption in check (in case of api change in future).